salt/modules/zypper.py

```diff
@@ -649,10 +649,10 @@ def version_cmp(ver1, ver2, ignore_epoch=False):
     return __salt__['lowpkg.version_cmp'](ver1, ver2, ignore_epoch=ignore_epoch)
 
 
-def list_pkgs(versions_as_list=False, **kwargs):
+def list_pkgs(versions_as_list=False, attr=None, **kwargs):
```

Comment:
> nix.py is new in development
...
> Which standard is going to be used attributes=True/False or attrib=[list of attributes]? 

This seems to me like a valid but different concern than the two discussed above (what the default behaviour should be and whether the new parameter should be in `**kwargs`). I am now restricting the discussion to this last topic.

The choice of the `attr` name and its format here was taken mainly to stay coherent with the existing [info_installed](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.zypper.html#salt.modules.zypper.info_installed) - lacking a broader overview I am open to change that if the Salt team indicates to do so.